tests/kohana/Log/FileTest.php

```diff
@@ -168,6 +168,39 @@ public function test_writer_does_not_create_existing_file_but_only_appends(array
 		$this->assertSame($expected_file_contents, $writer->get_written_logs());
 	}
 
+	/**
+	 * Provider for test_created_files_folders_permissions
+	 */
+	public function provider_created_files_folders_permissions()
+	{
+		return [
+			[0777, 0666, TRUE],
+			[0700, 0600, TRUE],
```

Comment:
I am not sure about this test case. I wonder if it fails when PHPUnit is run with different user in different CI environments?